### PR TITLE
Implement Runtime.evaluate

### PR DESCRIPTION
**Pull request by jeprojects**
Basic implementation of Runtime.evaluate based of `call_js_fn`

### CODE REVIEW

File: src/browser/tab/mod.rs [hunk 1]
```diff
@@ -663,6 +663,28 @@ impl<'a> Tab {
             .call_method(protocol::debugger::methods::GetScriptSource { script_id })?
             .script_source)
     }
+
+    /// Executes javascript
+    ///
+    /// See Devtools protocol doc: https://chromedevtools.github.io/devtools-protocol/tot/Runtime#method-evaluate
```
Comment (lukaslueg) [inline src/browser/tab/mod.rs]: The docs should not just refer to an external syntax.

File: src/browser/tab/mod.rs [hunk 2]
```diff
@@ -663,6 +663,28 @@ impl<'a> Tab {
             .call_method(protocol::debugger::methods::GetScriptSource { script_id })?
             .script_source)
     }
+
+    /// Executes javascript
```
Comment (lukaslueg) [inline src/browser/tab/mod.rs]: The first sentence from the original docs is "Evaluates expression on global object.", which is a much better description thatn "Executes javascript".

Comment (jeprojects) [reply src/browser/tab/mod.rs]: Thanks.. Made the changes

File: src/browser/tab/mod.rs (hunk 1 above)
Comment (jeprojects) [reply src/browser/tab/mod.rs]: Fixed

**Comment by atroche** [comment]
Cheers, good work

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Pull request merged**